controllers/finalizer.go

```diff
@@ -157,16 +157,14 @@ func removeFinalizer(name string, namespace string, gvrShort string, client k8sc
 				// broadcast event
 
 				// get resource for event broadcasting
-				r, err := client.Resource(schema.GroupVersionResource{
+				_, err := client.Resource(schema.GroupVersionResource{
```

Comment:
It seems like the purpose of the else clause in line 156 is to broadcast removal event. So, if you want to remove the broadcastEvent, it is the else clause that needs to be removed.

Reply:
> It seems like the purpose of the else clause in line 156 is to broadcast removal event. So, if you want to remove the broadcastEvent, it is the else clause that needs to be removed.

agree